src/unum/wireless/mt76xx_common/mt76xx_stas_platform.c

```diff
@@ -106,20 +106,22 @@ int wt_tpl_fill_sta_info(WT_JSON_TPL_RADIO_STATE_t *rinfo,
 
     // Check that the STA is connected to the VAP
     int vap_id = -1;
-    if(strncmp(vinfo->ifname, "rai", 3) == 0) {
+    if(strncmp(vinfo->ifname, WIFI_RADIO_5_PREFIX, 3) == 0) {
         vap_id = vinfo->ifname[3] - '0';
-    } else if(strncmp(vinfo->ifname, "ra", 2) == 0) {
+    } else if(strncmp(vinfo->ifname, WIFI_RADIO_24_PREFIX, 2) == 0) {
         vap_id = vinfo->ifname[2] - '0';
     }
     if(vap_id < 0 || vap_id > 3) {
         log("%s: failed to get VAP id from <%s>\n", __func__, vinfo->ifname);
         return -3;
     }
 
+#ifdef WT_QUERY_STAS_FOR_EACH_VAP
```

Comment:
Still need to figure out what's going on here. Moving my last comment from the old pull request here:
```
In this case your platform code must be reporting STAs incorrectly.  
You probably need to define WT_QUERY_STAS_FOR_EACH_VAP in your platform code.  
That will make mt76xx generic code query STAs from each VAP interface on a radio, for example ra0, ra1 and so on.  
Without that define mt76xx code expects the drivers to return STAs for all the VAP interfaces when querying interface 0, i.e. ra0 and after that filters them out by individual VAPs by checking the ApIdx.
```